mbq/metrics/__init__.py

```diff
@@ -119,16 +118,12 @@ def timing(self, metric, value, tags=None):
     def service_check(self, check_name, status, tags=None, message=None):
         # the dogstatsd client doesn't use namespace or constant_tags
         # for service_check but we want to be consistent
-
-        check_name = self._combine_metric(check_name)
-        if self.statsd.namespace:
-            check_name = self.statsd.namespace + '.' + check_name
-
-        tags = self._combine_tags(tags)
-        if self.statsd.constant_tags:
-            tags += self.statsd.constant_tags
-
-        self.statsd.service_check(check_name, status, tags=tags, message=message)
+        _statsd.service_check(
+            self._combine_metric(check_name),
```

Comment:
If I'm reading this right, the prefix wasn't being added to the service check name but now it will. That's going to break monitors.

Reply:
`prefix` _was_ being added to the service check name. The `_combine_metric` function adds the prefix.

Reply:
Ah I didn't the _combine_metric call above. Disregard!